reports/helmcharts/README.md

```diff
@@ -1,5 +1,72 @@
 # Helm Reports Microservice
+
 The Version Reporter Service MicroServices Project
 
-### Container image used
-https://hub.docker.com/r/alpine/k8s
+## Helm Charts
+
+This directory contains Helm charts used for deploying various services. Helm is a package manager for Kubernetes that allows you to define, install, and upgrade even the most complex Kubernetes applications.
```

Comment:
Think this could be updated to be more accurate?
This dir contains a script which fetches helm objects from a cluster by running as a job, and using helm whatup tells us the latest version and our deployed version etc etc
Examples of apps that this includes